addons/Popochiu/Engine/Others/PopochiuUtils.gd

```diff
@@ -1,31 +1,12 @@
 tool
 extends Node
+class_name PopochiuUtils
 # Utility functions for Popochiu
 # ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
 
 
-func get_screen_coords_for(node: Node) -> Vector2:
-	return node.get_viewport().canvas_transform * node.get_global_position()
-
-
-# Gets a random element from an Array
-func get_random_array_element(arr: Array):
-	randomize()
-	var idx := randi() % arr.size()
-
-	return arr[idx]
-
-
-# Gets a random index from an Array
-func get_random_array_idx(arr: Array) -> int:
-	randomize()
-	var idx := randi() % arr.size()
-
-	return idx
-
-
-# https://gist.github.com/me2beats/443b40ba79d5b589a96a16c565952419 ❱❱❱❱❱❱❱❱❱❱❱❱❱❱❱❱
-func snake2camel(string:String)->String:
+# https://gist.github.com/me2beats/443b40ba79d5b589a96a16c565952419 ❱❱❱❱❱❱❱❱❱❱❱❱
+static func snake2camel(string:String) -> String:
```

Comment:
ah damn I didn't notice you had those (Godot 4 has native ones btw), I could have used it instead of just lowering the case of tag names in the importer. Have we got time to address this before release?

Reply:
Of course. Do you want me to make the change?

Reply:
While you're at it... it's the only `tolower()` that's in the plugin folder I guess. It's in the function that creates animations. Changing lowercase to snake_case would be great. Porting it to Godot 4, we can use the new built-in functions for that.

It's not 100% important tho: this plugin will be ported to G4 pretty soon I hope :)